rio_tiler/main.py

```diff
@@ -44,32 +44,32 @@ def tile(address, tile_x, tile_y, tile_z, rgb=None, tilesize=256, nodata=None, a
         Mercator tile Y index.
     tile_z : int
         Mercator tile ZOOM level.
-    rgb : tuple, int, optional (default: (1, 2, 3))
-        Bands index for the RGB combination.
+    indexes : tuple, int, optional (default: (1, 2, 3))
+        Bands indexes for the RGB combination.
     tilesize : int, optional (default: 256)
         Output image size.
-    nodata: int or float, optional (defaults: None)
-    alpha: int, optional (defaults: None)
-        Force alphaband if not present in the dataset metadata
+    nodata: int or float, optional
+        Overwrite nodata value for mask creation.
+    alpha: int, optional
+        Overwrite alpha band index for mask creation.
 
     Returns
     -------
     data : numpy ndarray
     mask: numpy array
-    """
 
+    """
     with rasterio.open(address) as src:
         wgs_bounds = transform_bounds(
             *[src.crs, 'epsg:4326'] + list(src.bounds), densify_pts=21)
-        nodata = nodata if nodata is not None else src.nodata
```

Comment:
Not mentioned but, this a `big` change (more a bug fix) we should let `gdal/rasterio` handle internal nodata value (read_masks).